pkg/api/bzz.go

```diff
@@ -360,9 +360,10 @@ FETCH:
 					u.Path += "/"
 					redirectURL := u.String()
```

Comment:
do we really need this `u` variable? It looks like we could just log `url` parameter as `r.URL`, right?